### PR TITLE
clarify the usage of the value field in PodChaos

### PR DESCRIPTION
**Pull request by silentred**
add the usage of value in PodChaos to the document.

### CODE REVIEW

**Comment by cwen0** [comment]
@silentred Thanks for your contributions!

File: doc/pod_chaos.md [hunk 1]
```diff
@@ -39,6 +40,7 @@ Description:
 
 * **action** defines the specific chaos action for the pod. In this case, it is pod failure.
 * **mode** defines the mode to run chaos action. Supported mode: `one / all / fixed / fixed-percent / random-max-percent`.
+* **value** depends on the value of `mode`. If `mode` is `one` or `all`, leave `value` empty. If `fixed`, provide an integer of pods to do chaos action. If `fixed-percent`, provide a number from 0-100 to specify the max % of pods the server can do chaos action. If `random-max-percent`, provide a number from 0-100 to specify the % of pods to do chaos action
```
Comment (fewdan) [inline doc/pod_chaos.md]: " the max % of " and " the % of " looks strange ,  is it inappropriate?

Comment (silentred) [reply doc/pod_chaos.md]: It was copied from PodChaos field comment. I have changed it to percent.

File: doc/pod_chaos.md [hunk 2]
```diff
@@ -39,6 +40,7 @@ Description:
 
 * **action** defines the specific chaos action for the pod. In this case, it is pod failure.
 * **mode** defines the mode to run chaos action. Supported mode: `one / all / fixed / fixed-percent / random-max-percent`.
+* **value** depends on the value of `mode`. If `mode` is `one` or `all`, leave `value` empty. If `fixed`, provide an integer of pods to do chaos action. If `fixed-percent`, provide a number from 0-100 to specify the max percent of pods the server can do chaos action. If `random-max-percent`, provide a number from 0-100 to specify the percent of pods to do chaos action
```
Comment (cwen0) [inline doc/pod_chaos.md]: If `fixed-percent`, provide a number from 0-100 to specify the percent of pods the server can do chaos action. If `random-max-percent`, provide a number from 0-100 to specify the max percent of pods to do chaos action.   

In addition, Can you help to update PodChaos field comment?

Comment (silentred) [reply doc/pod_chaos.md]: The change has been made.

**Comment by cwen0** [comment]
@fewdan @YangKeao PTAL

**Pull request merged**